components/io_expander/esp_io_expander_tca9554/idf_component.yml

```
@@ -0,0 +1,7 @@
+version: "1.0.0"
+description: ESP IO Expander - tca9554
+url: https://github.com/espressif/esp-bsp/tree/master/components/io_expander/esp_io_expander_tca9554
+dependencies:
+  idf: ">=4.4.2"
+  esp_io_expander:
+    version: "^1.0"
```

```suggestion
    version: "^1.0.1"
```

We will delete v1.0.0 of esp_io_expander